src/pages/components/Button.tsx

```diff
@@ -0,0 +1,8 @@
+import styles from '@/src/styles/Button.module.css'
+
+function Button({ className = 'small', children = '' }) {
+  const classNames = `${styles.Button} ${styles[className]}`
+  return <button className={classNames}>{children}</button>

```

Comment:
```suggestion
interface ButtonProps extends React.ButtonHTMLAttributes<HTMLButtonElement>  {
  size?: 'small' | 'small48' | 'medium' | 'large'
}

function Button({ size = 'small', className, children, ...props }: ButtonProps) {
  const classNames = `${styles.Button} ${styles[size]} ${className}`
  return <button className={classNames} {...props}>{children}</button>
```

우선 새로 생성한 Button 컴포넌트에 기존 button element와 특별하게 다른 props를 사용하고 있지않아서 이렇게 해주면 더 공통적으로 사용하기 좋을 것 같습니다!

또한 size를 따로 두어서 size를 따로 두어서 조금 더 사용성이 좋게 할 수 있고요! (size 의 타입은 새로 선언해도 좋구요!)

정답은 없지만 조금 더 고도화를 하나씩 해보는 연습을 하다보면 익숙해질 거에요!